tests/e2e/fixtures/snackbarError.js

```diff
@@ -5,7 +5,12 @@ const SnackBarError = {
   FROZEN_CARD: 'That card is frozen! You must wait a turn to play it',
   NOT_IN_GAME: "You can't do that because you're not a player in this game",
   GAME_IS_FULL: `Cannot join that game because it's already full`,
-  CANT_FIND_GAME: "Can't find game"
+  CANT_FIND_GAME: "Can't find game",
+  ONE_OFF: {
+    THREE_EMPTY_SCRAP: 'You can only play a 3 as a one-off if there are cards in the scrap pile',
+    FOUR_EMPTY_HAND: 'You cannot play a 4 as a one-off while your opponent has no cards in hand',
+    EMPTY_DECK: "You can't play that one-off unless there are cards in the deck",
+  }
```

Comment:
Updated to add additional errors that the tests can look for via fixtures instead of hard coding in the tests. It would be nice to reconcile this more concretely with the language files themselves, but I think that's an issue for another update